hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/dao/GatewayServiceLogEventConverter.java

```diff
@@ -0,0 +1,14 @@
+package org.hypertrace.core.graphql.log.event.dao;
+
+import io.reactivex.rxjava3.core.Single;
+import org.hypertrace.core.graphql.common.request.ResultSetRequest;
+import org.hypertrace.core.graphql.log.event.schema.LogEventResultSet;
+import org.hypertrace.gateway.service.v1.log.events.LogEventResponse;
+
+
+public class GatewayServiceLogEventConverter {
```

Comment:
nit: obviously no implementation yet, but just as a heads up we try to restrict visibility to minimum possible in most of the GQL code - usually that means classes are packaged protected and only interfaces are public (and only exist if needed)